apisix/cli/env.lua

```diff
@@ -28,6 +28,12 @@ local package = package
 
 
 return function (apisix_home, pkg_cpath_org, pkg_path_org)
+    -- ulimit setting should be checked when APISIX starts
+    ulimit = tonumber(util.trim(util.execute_cmd("ulimit -n")))
```

Comment:
Should check the `nil, err` returned from execute_cmd, and log the err

Reply:
Got it